src/main/java/com/mentorship/vineservice/domain/OrderVine.java

```diff
@@ -0,0 +1,23 @@
+package com.mentorship.vineservice.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.EmbeddedId;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "order_vine")
+@Getter
+@Setter
+@NoArgsConstructor
+public class OrderVine {
```

Comment:
why did you decide not to map explicitly order and wine in this entity?

Reply:
My mistake, changed